cupy/core/core.pyx

```diff
@@ -2922,11 +2945,26 @@ cpdef ndarray tensordot_core_v11(
     cdef cuDoubleComplex one_D, zero_D
     cdef size_t one_ptr, zero_ptr
 
+    if get_fp32_compute_type() == FP32_COMPUTE_TBD:
+        fp32_compute_type = FP32_COMPUTE_DEFAULT
+        if int(os.getenv('CUPY_TF32', '0')) > 0:
+            fp32_compute_type = FP32_COMPUTE_FAST_TF32
+        set_fp32_compute_type(fp32_compute_type)
+
     cdef int compute_type
```

Comment:
I notice Cython throws a warning at this line:
```
    cupy/core/core.cpp: In function ‘__pyx_obj_4cupy_4core_4core_ndarray* __pyx_f_4cupy_4core_4core_tensordot_core_v11(Py_ssize_t, Py_ssize_t, Py_ssize_t, Py_ssize_t, Py_ssize_t, __pyx_obj_4cupy_4core_4core_ndarray*, Py_ssize_t, __pyx_obj_4cupy_4core_4core_ndarray*, Py_ssize_t, __pyx_obj_4cupy_4core_4core_ndarray*, Py_ssize_t, int)’:
    cupy/core/core.cpp:48655:13: warning: ‘__pyx_v_compute_type’ may be used uninitialized in this function [-Wmaybe-uninitialized]
       __pyx_t_1 = __pyx_f_13cupy_backends_4cuda_4libs_6cublas_gemmEx(__pyx_v_handle, ((int)__pyx_v_transa), ((int)__pyx_v_transb), ((int)__pyx_v_m), ((int)__pyx_v_n), ((int)__pyx_v_k), __pyx_v_one_ptr, __pyx_v_a->data->ptr, __pyx_v_a_cuda_dtype, ((int)__pyx_v_lda), __pyx_v_b->data->ptr, __pyx_v_b_cuda_dtype, ((int)__pyx_v_ldb), __pyx_v_zero_ptr, __pyx_v_c->data->ptr, __pyx_v_c_cuda_dtype, ((int)__pyx_v_ldc), __pyx_v_compute_type, __pyx_v_algo, 0); if (unlikely(!__pyx_t_1)) __PYX_ERR(0, 2972, __pyx_L1_error)
       ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Could you initialize it to an invalid value? Something like:
```suggestion
    cdef int compute_type = -1
```

Reply:
Thanks!  I addressed it a different way, but your suggested way seems better.